client/src/components/Navigation.jsx

```diff
@@ -2,8 +2,8 @@
 
 import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
-import { Layout, Drawer, Button, Menu } from 'antd';
-import { MenuOutlined } from '@ant-design/icons';
+import { Layout, Drawer, Button, Menu, Select } from 'antd';
+import { MenuOutlined, createFromIconfontCN } from '@ant-design/icons';
```

Comment:
If we're not using `createFromIconfontCN` maybe remove from PR just to avoid the lint warning.